yjit/src/backend/arm64/mod.rs

```diff
@@ -188,6 +188,12 @@ fn emit_load_value(cb: &mut CodeBlock, rd: A64Opnd, value: u64) -> usize {
 /// These are caller-saved registers.
 pub static TEMP_REGS: [Reg; 5] = [X1_REG, X9_REG, X10_REG, X14_REG, X15_REG];
 
+#[derive(Debug, PartialEq)]
+enum EmitError {
+    RetryOnNextPage,
+    OutOfMem,
```

Comment:
nits: How about just `OutOfMemory`? Generally it's easier to understand a name if you don't abbreviate it, and this one would be still short enough (at least shorter than `RetryOnNextPage`).